src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/ExplosivePickaxe.java

```diff
@@ -63,11 +67,18 @@ public boolean onBlockBreak(BlockBreakEvent e, ItemStack item, int fortune, List
                                         continue;
                                     }
 
-                                    Block b = e.getBlock().getRelative(x, y, z);
-                                    breakBlock(e.getPlayer(), item, b, fortune, drops);
+                                    blocks.add(e.getBlock().getRelative(x, y, z));
                                 }
                             }
                         }
+
+                        EntityExplodeEvent entityExplodeEvent = new EntityExplodeEvent(e.getPlayer(), e.getBlock().getLocation(), blocks, 0);
+                        Bukkit.getServer().getPluginManager().callEvent(entityExplodeEvent);
+                        if (!entityExplodeEvent.isCancelled()) {
+                            entityExplodeEvent.blockList().forEach(b -> {
```

Comment:
Shouldn't use curly brace for lambda with one line.

Reply:
IDE completing! Ill change ;)